app/src/main/java/com/example/healthinspector/Fragments/ProductFinderFragment.java

```diff
@@ -1,20 +1,14 @@
 package com.example.healthinspector.Fragments;
 
-import android.Manifest;
 import android.annotation.SuppressLint;
-import android.content.Context;
-import android.content.pm.PackageManager;
 import android.location.Location;
 import android.location.LocationManager;
 import android.os.Bundle;
 
 import androidx.activity.result.ActivityResultLauncher;
-import androidx.activity.result.contract.ActivityResultContracts;
```

Comment:
Good call removing unused imports! Check out this link to make this automatic - https://fburl.com/w5f21d5d